main/src/main/java/cgeo/geocaching/log/CacheLogsViewCreator.java

```diff
@@ -26,6 +26,8 @@
 import androidx.annotation.NonNull;
 import androidx.appcompat.widget.TooltipCompat;
 
+import com.google.android.material.chip.Chip;
```

Comment:
This is the Checkstyle error:
~~~
           <property name="groups" value="cgeo,android,androidx,java,javax,*" />

~~~
 So, the `com` is wrong between `androidx` and `java`.

Reply:
strange, i didn't touch any of the import settings in intellij, so why would it be wrong? Anyway, pushed again now, let's hope this fixes it.

Reply:
IntelliJ/AS has the habit of auto-inserting new imports in places different to what our CeckStyle rules expect - not sure if that could be synced - @bekuno?

Reply:
Would be great if AS could pick this up automatically from checkstyle config somehow. Is there a way?

I had to set c:geo's import rules manually in AS config. (c:geo rules are strangely different from any other project I work in)

Reply:
We have in the project the file `<cgeo>\.idea\checkstyle-idea.xml`.
The default value is set to "cgeo-checks".
If you don't change this and add to your personally git ignore file, then you have this active.
The AS checkstyle should then use this configuration.

Reply:
> > Would be great if AS could pick this up automatically from checkstyle config somehow. Is there a way?
> 
> We have in the project the file `<cgeo>\.idea\checkstyle-idea.xml`.
> The default value is set to "cgeo-checks".
> If you don't change this and add to your personally git ignore file, then you have this active.
> The AS checkstyle should then use this configuration.

I was referring to the AS import optimize setting (Settings, Editor, Code Style, Java, Imports, Import Layout). The ones @ztNFny referred to above

Reply:
> > > Would be great if AS could pick this up automatically from checkstyle config somehow. Is there a way?
> > 
> > 
> > We have in the project the file `<cgeo>\.idea\checkstyle-idea.xml`.
> > The default value is set to "cgeo-checks".
> > If you don't change this and add to your personally git ignore file, then you have this active.
> > The AS checkstyle should then use this configuration.
> 
> I was referring to the AS import optimize setting (Settings, Editor, Code Style, Java, Imports, Import Layout). The ones @ztNFny referred to above

We discussed it in the meeting today. See Issue #17287